### PR TITLE
DBZ-1547 02/2019 Newletter

### PR DESCRIPTION
**Pull request by Naros**
https://issues.jboss.org/browse/DBZ-1547

### CODE REVIEW

File: blog/2019-10-17-debezium-newsletter-02-2019.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+questions that are relevant to our user community.
+
+== Upcoming Events
+
+* https://jokerconf.com[JokerConf - Practical change data streaming uses cases with Apache Kafka and Debezium]
+* https://qconsf.com[QCon San Fransisco - Practical change data streaming use cases with Apache Kafka and Debezium]
+
+== Articles
+
+This https://debezium.io/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[recent blog] by Gunnar Morling discusses how Debezium combined with Kafka stream post processing can deliver an enriched stream of events suitable for even the most complicated of tasks like audit tracking.
```
Comment (gunnarmorling) [inline blog/2019-10-17-debezium-newsletter-02-2019.adoc]: Could you add an introductory sentence like "There were a lot of blog posts touching on Debezium lately; here are some of the latest posts you shouldn't miss:". Then a list with the ones you already got and also these:

* https://speakerdeck.com/jbfletch/using-kafka-to-discover-events-hidden-in-your-database
* https://medium.com/convoy-tech/logs-offsets-near-real-time-elt-with-apache-kafka-snowflake-473da1e4d776
* https://info.crunchydata.com/blog/postgresql-change-data-capture-with-debezium

And then add a sentence with the link to the online-resources page.

File: blog/2019-10-17-debezium-newsletter-02-2019.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Rod Shokrian from Varo https://medium.com/engineering-varo/event-driven-architecture-and-the-outbox-pattern-569e6fba7216[recently blogged] about their CDC solution and experiences using Debezium in conjunction with the Outbox Pattern.
+
+Community involvement is critical to each Open Source project and Debezium is no different.  Joy Gao https://wecode.wepay.com/posts/streaming-cassandra-at-wepay-part-1[blogs] about her experience at WePay where CDC innovation brought Cassandra and Debezium together.
+
+== Examples
+
+* [NEW] https://github.com/debezium/debezium-examples/tree/master/auditlog[Build Audit Logs using Debezium and Kafka Streams]
```
Comment (gunnarmorling) [inline blog/2019-10-17-debezium-newsletter-02-2019.adoc]: An introductory sentence could be nice: "We've added and updated..."

File: blog/2019-10-17-debezium-newsletter-02-2019.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+== Questions and answers
+
+* https://stackoverflow.com/questions/58337376/debezium-postgres-error-parameter-include-unchanged-toast-was-deprecated[Debezium postgres ERROR: Parameter "include-unchanged-toast" was deprecated]
+* https://stackoverflow.com/questions/57095096/unable-to-register-debezium-kafka-connect-connector-in-ssl-enabled-kafka-clust[Unable to register Debezium (Kafka-Connect) connector in SSL enabled Kafka cluster]
+* https://groups.google.com/d/msg/debezium/1vKTWwcf71I/L98pV5nnBgAJ[Debezium or database triggers]
+
+== Using Debezium?
+
+We recently added a https://www.debezium.io/community/users[community users] page to https://www.debezium.io.
+If you are a user of Debezium and would like to be included, please send us a GitHub Pull Request or reach out to us directly.
```
Comment (gunnarmorling) [inline blog/2019-10-17-debezium-newsletter-02-2019.adoc]:
```suggestion
If you are a user of Debezium and would like to be included, please send us a GitHub pull request or reach out to us directly.
```

File: blog/2019-10-17-debezium-newsletter-02-2019.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+:awestruct-tags: [ community, news, newsletter ]
+:awestruct-layout: blog-post
+
+Welcome to the Debezium community newsletter in which we share blog posts, group discussions, as well as StackOverflow
+questions that are relevant to our user community.
+
+== Upcoming Events
+
+* https://jokerconf.com[JokerConf - Practical change data streaming uses cases with Apache Kafka and Debezium]
+* https://qconsf.com[QCon San Fransisco - Practical change data streaming use cases with Apache Kafka and Debezium]
```
Comment (gunnarmorling) [inline blog/2019-10-17-debezium-newsletter-02-2019.adoc]: Here's the deep links: 

* https://jokerconf.com/en/2019/talks/6nkqqv8zj4gmujxmo64ov3/
* https://qconsf.com/sf2019/presentation/practical-change-data-streaming-use-cases-apache-kafka-debezium

**Comment by gunnarmorling** [comment]
Very nice; a few comments inline. If you got those covered, could you merge it yourself? Thanks a lot, @Naros!

**Comment by Naros** [comment]
@gunnarmorling Will do, I'll publish this on Thursday 10/17 as planned.

**Comment by gunnarmorling** [comment]
@Naros, here's one more thing which would be nice to add, our internship opening: https://twitter.com/debezium/status/1184514850627739649

**Comment by Naros** [comment]
Applied

**Pull request merged**